tests/tasks/Cloudwatch/cw-metric.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name:  cloudwatch
+  namespace: tekton-pipelines
+spec:
+  description: "cw task to push metrics relying on generate task."
+  params:
+    - name: region
+      default: "us-west-2"
+    - name: desired-nodes
+    - name: load-result
+  steps:
+  - name: cw-emit
+    image: alpine/k8s:1.22.6
+    script: |
+      if [[ "$(params.load-result)" -eq 0 ]]; then
+      aws cloudwatch --region $(params.region) put-metric-data --metric-name $(params.desired-nodes)_nodes --namespace Cl2Load-Tests --unit Count --value 1 --dimensions TypeOfTest=Cl2_Test
```

Comment:
How does it know which AWS account to use?

Reply:
I believe whatever the aws creds that we were using o run up these pipelines ,it will consider the same account ,unless if we want specify any.